### PR TITLE
Use official postgres image, fix createAndRunJob.spec.ts

### PR DESCRIPTION
**Pull request by j16r**
* Wake up StatsPusher outside of callbacks, so the transaction should be committed and the events immediately available for syncing
 * Switch to official postgres docker image
 * Remove debug logs from resumeRunsOnNewHeadWorker

[Fixes #169991336] ?

### CODE REVIEW

**Comment by j16r** [comment]
This looks like an easy win, mind giving us a sample of some of the metrics, perhaps the output of `/metrics` when using this?

File: core/services/run_manager.go [hunk 1]
```diff
@@ -389,13 +396,16 @@ func (jm *runManager) updateWithError(run *models.JobRun, msg string, args ...in
 		logger.Errorw("Error saving run", run.ForLogger("error", err)...)
 		return err
 	}
+	jm.statsPusher.PushNow()
 	return nil
 }
 
 func (jm *runManager) updateAndTrigger(run *models.JobRun) error {
+	defer jm.statsPusher.PushNow()
```
Comment (HenryNguyen5) [inline core/services/run_manager.go]: Not related to this PR but, what's the difference between a `JobManager` (jm) and a `RunManager` (the type itself)?

File: core/services/synchronization/stats_pusher.go [hunk 2]
```diff
@@ -26,8 +26,18 @@ var (
 	})
 )
 
-// StatsPusher polls for events and pushes them via a WebSocketClient
-type StatsPusher struct {
+//go:generate mockery -name StatsPusher -output ../../internal/mocks/ -case=underscore
```
Comment (HenryNguyen5) [inline core/services/synchronization/stats_pusher.go]: Should we leave a little comment here saying that this is to generate the `StatsPusher` mock? I think something like this belongs in a README of some sort more than anything.

File: core/services/synchronization/stats_pusher.go [hunk 3]
```diff
@@ -26,8 +26,18 @@ var (
...
+
+// statsPusher polls for events and pushes them via a WebSocketClient
```
Comment (HenryNguyen5) [inline core/services/synchronization/stats_pusher.go]: Should we give some examples of what types of events are pushed? And what types of clients would consume these events?

Comment (HenryNguyen5) [reply core/services/synchronization/stats_pusher.go]: Also seems to be a typo, should be uppercase `StatsPusher`

File: core/services/synchronization/stats_pusher.go [hunk 4]
```diff
@@ -224,13 +243,11 @@ func createSyncEventWithStatsPusher(sp *StatsPusher, orm *orm.ORM) func(*gorm.Sc
 		event := models.SyncEvent{
 			Body: string(bodyBytes),
 		}
-		err = scope.DB().Save(&event).Error
+		err = scope.DB().Create(&event).Error
```
Comment (HenryNguyen5) [inline core/services/synchronization/stats_pusher.go]: What's the difference between `Create` and `Save` here?

File: core/services/synchronization/stats_pusher.go [hunk 5]
```diff
@@ -224,13 +243,11 @@ func createSyncEventWithStatsPusher(sp *StatsPusher, orm *orm.ORM) func(*gorm.Sc
...
 		if err != nil {
 			scope.Err(errors.Wrap(err, "createSyncEvent#Save failed"))
```
Comment (HenryNguyen5) [inline core/services/synchronization/stats_pusher.go]:
```suggestion
			scope.Err(errors.Wrap(err, "createSyncEvent#Create failed"))
```

File: core/services/run_manager.go (hunk 1 above)
Comment (j16r) [reply core/services/run_manager.go]: There is no JobManager, this is just a relic.

File: core/services/synchronization/stats_pusher.go (hunk 4 above)
Comment (j16r) [reply core/services/synchronization/stats_pusher.go]: Create only creates new records, Save can update a record if it already exists - which it never should.

File: core/services/synchronization/stats_pusher.go (hunk 2 above)
Comment (j16r) [reply core/services/synchronization/stats_pusher.go]: https://github.com/smartcontractkit/chainlink/pull/2229#pullrequestreview-349730923

**Pull request merged**